invoice/epc_qr.py

```diff
@@ -0,0 +1,95 @@
+"""EPC QR code generation utilities."""
+
+from decimal import Decimal
+from typing import Optional
+
+from schwifty import BIC, IBAN
+
+
+def gen_epc_qr_data(beneficiary_name: str, beneficiary_iban: str | IBAN, beneficiary_bic: Optional[str | BIC] = None,
```

Comment:
We have the IBAN as object in the DB. No need to pass a string. We should pass the IBAN object and get the string in the function.
We can even thing of not passing the BIC.

Reply:
IBAN/BIC extend str and thus are allowed to be passed in here ;)